package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "pharmacy-data-etl",
-  "version": "0.2.0",
+  "version": "0.3.0",
   "description": "ETL to extract data from Syndication/Organisations and store as JSON",
   "main": "app.js",
   "scripts": {
     "coverage-check": "istanbul check-coverage --config .istanbul.yml",
-    "coverage-generate": "yarn istanbul -- cover _mocha -- --recursive test",
+    "coverage-generate": "yarn istanbul cover -- _mocha -- --recursive test",
```

Comment:
It's strange that yarn warns about not needing the '--', but if you take them out it doesn't work on this line